Client/Components/DashBoard.razor.cs

```diff
@@ -21,12 +21,14 @@ public partial class DashBoard
         
         DateOnly date = new DateOnly(2022, 1, 1);
         private IList<string> dataMetrics = new List<string>();
```

Comment:
Should probably start with underscore to match other private variables

Reply:
I think we need to agree on whether or not to use underscores for private variables. Is it ok to change someone else's code?

Reply:
That's ok. "Private instance fields start with an underscore (_) and the remaining text is camelCased" according to [Microsoft's naming conventions for C#](https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/identifier-names).